fdbserver/DataDistribution.actor.cpp

```diff
@@ -601,6 +570,10 @@ struct DataDistributor : NonCopyable, ReferenceCounted<DataDistributor> {
 			remoteDcIds.push_back(regions[1].dcId);
 		}
 	}
+
+	Future<Void> waitDataDistributorEnabled(const DDEnabledState* ddEnabledState) const {
```

Comment:
Would suggest continuing to call this waitForDataDistributionEnabled, as it still has to do with the overall role rather than the specific struct that is encapsulating the method

Reply:
Finally the dataDistributor ACTOR will become a `run` method within this class